Integrations/integration-QRadar.yml

```diff
@@ -42,296 +38,554 @@ configuration:
-  name: incidentType
-  defaultvalue: ""
-  type: 13
-  required: false
 - display: Use system proxy settings
   name: proxy
   defaultvalue: "true"
   type: 8
   required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
 script:
   script: |-
-    // The command input arg holds the command sent from the user.
-    var username = params.credentials ? params.credentials.identifier : '';
-    var password = params.credentials ? params.credentials.password : '';
-    var token = params.token;
-    if (!token && !password) {
-        throw 'Either credentials or auth token should be provided.'
-    }
-    var server = params.server+'/';
-    var insecure = params.insecure;
-    var query = params.query;
-    var offensesPerCall = parseInt(params.offensesPerCall || 50);
-    offensesPerCall = offensesPerCall > 50 ? 50 : offensesPerCall;
-
-    var createIncidentFromOffense = function(incident) {
-        var keys = Object.keys(incident);
-        var labels = [];
-        var occured = (new Date(incident.start_time)).toISOString();
-        for (var i = 0; i<keys.length; i++) {
-            labels.push({'type': keys[i], 'value': String(incident[keys[i]])});
-        }
+    import sys, os, requests, traceback, datetime, json
+    from requests.exceptions import HTTPError
+
+    ''' GLOBAL VARS '''
+    SERVER = demisto.params()['server'][:-1] if demisto.params()['server'].endswith('/') else demisto.params()['server']
+    USERNAME = demisto.params().get('credentials').get('identifier')
+    PASSWORD = demisto.params().get('credentials').get('password')
+    TOKEN = demisto.params().get('token')
+    USE_SSL = not demisto.params().get('insecure', False)
+    CUSTOM_HEADERS = demisto.args().get('headers')
+    AUTH_HEADERS = {'SEC': str(TOKEN), 'Content-Type':  'application/json'}
+    OFFENSES_PER_CALL = int(demisto.params().get('offensesPerCall', 50))
+
+    OFFENSES_PER_CALL = 50 if OFFENSES_PER_CALL > 50 else OFFENSES_PER_CALL
+
+    if not TOKEN and not PASSWORD:
+        raise Exception('Either credentials or auth token should be provided.')
+
+    if CUSTOM_HEADERS:
+        CUSTOM_HEADERS = CUSTOM_HEADERS.split(',')
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    # Header names transformation maps
+    # Format: {'OldName': 'NewName'}
+
+    OFFENSES_NAMES_MAP = {'follow_up': 'Followup', 'id': 'ID', 'description': 'Description', 'source_address': 'SourceAddress', 'destination_address': 'DestinatioAddress', 'start_time': 'StartTime', 'event_count': 'EventCount', 'magnitude': 'Magnitude', 'last_updated_time': 'LastUpdatedTime', 'offense_type': 'OffenseType'}
```

Comment:
Format the json/dict `OFFENSES_NAMES_MAP` and `SINGLE_OFFENSE_NAMES_MAP`, all of them

Reply:
between two variables add 1 or 2 spaces, this way it will be easier to read